### ISSUE TITLE
No module named 'requests'

### ISSUE THREAD

**Issue by PensiveProgrammer64** [BigBoiCJ/SteamAutoCracker#16]
Alright so maybe I am an idiot, but I installed this by going to "Code" and downloaded the zip file, unzipped it and then ran the "steam_auto_cracker.py" file and this error came up, the issue persisted after going to command prompt and using pip to install the 'request' module.

Traceback (most recent call last):
  File "C:\Users\Austin Polley\Downloads\SteamAutoCracker-main\steam_auto_cracker.py", line 4, in <module>
    import requests
ModuleNotFoundError: No module named 'requests'

**Comment by BigBoiCJ** [comment]
Probably caused by you having multiple Python installations. Had the same problem for a long time.

If you're running using the `py` command, try `py -m pip install requests`

**Comment by PensiveProgrammer64** [comment]
Thank you this worked well, however I have encountered another issue :( this time it says that it is missing the module win32api that is imported in the get_file_version file, I have tried the trick you just taught me however I keep getting this error:

`ERROR: Could not find a version that satisfies the requirement win32api (from versions: none)
ERROR: No matching distribution found for win32api`

Sorry to bother you again but I can't seem to figure this one out on my own, and if you need to know, I am running an official version of windows 10 standard that has not been altered in any way (to my knowledge).

**Comment by BigBoiCJ** [comment]
You should check this: https://pypi.org/project/pywin32/

The module is pywin32, and contains a lot of smaller modules, like win32api. It is pretty hard to install though, and I had trouble fixing those bugs. I hope it'll work for you!

**Comment by PensiveProgrammer64** [comment]
This was the final issue I had with your program and now it works properly, Thank you for being so kind and patient with me :)

**Comment by BigBoiCJ** [comment]
Glad we managed to solve the problems, and sorry for not documenting the requirements enough.

I should take some time for that but damn I'm lazy :(

**Comment by lolinator53** [comment]
Hey there, as a good rule of thumb, always use a [virtual environment](https://docs.python.org/3/library/venv.html) when working with different python versions or having dependencies on external libraries. This way you won‘t clog your local python installation with different external libraries and you can easily change the python version you are working with. IDEs like JetBrains [PyCharm](https://www.jetbrains.com/pycharm/) create one automatically.

**Comment by BigBoiCJ** [comment]
> Hey there, as a good rule of thumb, always use a [virtual environment](https://docs.python.org/3/library/venv.html) when working with different python versions or having dependencies on external libraries. This way you won‘t clog your local python installation with different external libraries and you can easily change the python version you are working with. IDEs like JetBrains [PyCharm](https://www.jetbrains.com/pycharm/) create one automatically.

True but it takes a bit of time to set up and understand. I do use virtual envs to 'compile' SAC to avoid bundling too many useless modules, but I have to admit, for most small scripts I do and use, I don't.

I should consider getting an IDE but I don't like how most want to associate with every possible file extension known to man. Already had problems with IDEs downloading multiple installations of python. At one point, I had 3 installed at once and it got pretty buggy. I nuked everything and now I just don't want to go back to that bs 😆

**Comment by lolinator53** [comment]
> I should consider getting an IDE

Are you telling me you are developing in IDLE??

> I don't like how most want to associate with every possible file extension known to man. Already had problems with IDEs downloading multiple installations of python.

Well, that's stuff that doesn't happen with PyCharm, it'll use your existing python installation (if none found, will ask to install or "show" where your python is) and create a venv for every project you have going with it, so no python installations will ever make it to your root fs (unless you explicitly tell it to). Also it has zero file associations (I couldn't find any) with it's default installation. 

In conclusion, you should really give it a try

**Comment by BigBoiCJ** [comment]
> Are you telling me you are developing in IDLE??

I use Notepad++

> In conclusion, you should really give it a try

You sold it pretty well, I'll try it.
